vimeo-networking/src/main/java/com/vimeo/networking/model/Recommendation.java

```diff
@@ -0,0 +1,73 @@
+package com.vimeo.networking.model;
+
+import com.vimeo.stag.GsonAdapterKey;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.io.Serializable;
+
+/**
+ * A model that contains recommendations for a User to follow.
+ * A recommendation can either be a {@link User} or {@link Channel}
+ * <p/>
+ * Created by zetterstromk on 8/15/16.
+ */
+public class Recommendation implements Serializable {
+
+    private static final long serialVersionUID = -1451431453348153582L;
+
+    static final String TYPE_CHANNEL = "channel";
+    static final String TYPE_USER = "user";
+
+    public enum RecommendationType {
+        NONE,
+        CHANNEL,
+        USER
+    }
+
+    /**
+     * Null if not recommended by a category
+     */
+    @Nullable
+    @GsonAdapterKey("category")
+    public Category mCategory;
+
+    @Nullable
+    @GsonAdapterKey("type")
+    public String mRecommendationType;
```

Comment:
Minor, could use a StringDef to enforce correct assignment... although since this will be set by the server response it most likely won't do much good